google/cloud/bigtable/internal/instance_admin.cc

```diff
@@ -175,6 +175,20 @@ btproto::AppProfile InstanceAdmin::GetAppProfile(
       "InstanceAdmin::GetAppProfile", status, true);
 }
 
+::google::longrunning::Operation InstanceAdmin::UpdateAppProfile(
+    bigtable::InstanceId instance_id, bigtable::AppProfileId profile_id,
+    AppProfileUpdateConfig config, grpc::Status& status) {
+  auto request = config.as_proto_move();
+  request.mutable_app_profile()->set_name(
+      InstanceName(instance_id.get() + "/appProfiles/" + profile_id.get()));
+
+  // This API is not idempotent, call it without retry.
```

Comment:
Which aspect of `UpdateAppProfile()` is non-idempotent? Is it the part about warnings for some changes? Or that the warnings received depend on state unrelated to the app profile itself?